common/sagas/config/node.ts

```diff
@@ -160,26 +160,29 @@ export function* handleNodeChangeIntent({
     );
   }
 
+  const isOffline = yield select(getOffline);
   if (isAutoNode(nodeIdToSwitchTo)) {
     shepherd.auto();
     if (getShepherdNetwork() !== nextNodeConfig.network) {
       yield apply(shepherd, shepherd.switchNetworks, [nextNodeConfig.network]);
     }
   } else {
     try {
-      yield apply(shepherd, shepherd.manual, [nodeIdToSwitchTo, false]);
+      yield apply(shepherd, shepherd.manual, [nodeIdToSwitchTo, isOffline]);
     } catch (err) {
       console.error(err);
       return yield* bailOut(translateRaw('ERROR_32'));
     }
   }
 
-  let currentBlock;
+  let currentBlock = '???';
```

Comment:
Hm, maybe better to keep the current block as a number, since once we introduce stricter typing to sagas this'll throw. Instead could set it to `-1` and display on the UI side `???` if the block number is negative

Reply:
Block number is a string in the reducer, the actions all take in a string.

Reply:
https://github.com/MyCryptoHQ/MyCrypto/blob/develop/common/reducers/config/meta/meta.ts#L8

https://github.com/MyCryptoHQ/MyCrypto/blob/develop/common/actions/config/actionCreators.ts#L113

Reply:
whoops, my bad

Reply:
No worries, might be worth refactoring at some point if we ever need to treat the block number as a number.